stable/dask-distributed/templates/dask-scheduler-deployment.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "scheduler-fullname" . }}
+  labels:
+    heritage: {{.Release.Service | quote }}
+    release: {{.Release.Name | quote }}
+    chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+    component: "{{.Release.Name}}-{{.Values.Scheduler.Component}}"
+spec:
+  replicas: {{default 1 .Values.Scheduler.Replicas}}
```

Comment:
Remove default. It comes from `values.yaml` and is thus redundant. Apply everywhere.